flow-typed/@testing-library/react-hooks.flow.js

```diff
@@ -0,0 +1,152 @@
+// @flow
+/**
+ * This file is based on:
+ * - https://github.com/testing-library/react-hooks-testing-library/blob/main/src/types/index.ts
+ * - https://github.com/testing-library/react-hooks-testing-library/blob/main/src/types/react.ts
+ * - https://github.com/testing-library/react-hooks-testing-library/blob/main/src/pure.ts
+ */
+ declare module "@testing-library/react-hooks" {
```

Comment:
This doesn't provide types for `@testing-library/react-hooks/server`.  I'm going to punt on that for now.